provider/kubernetes/kubernetes.go

```diff
@@ -188,6 +188,103 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 		}
 		templateObjects.TLS = append(templateObjects.TLS, tlsSection...)
 
+		if i.Spec.Backend != nil {
+			// If a global backend is configured, add this as a "default route" by adding a wildcard frontend, with the priority of the ingress.
+			defaultFrontendName := "global-default-frontend"
+			defaultBackendName := "global-default-backend"
+			if _, exists := templateObjects.Frontends[defaultFrontendName]; !exists {
+				// Ensure that we are not duplicating the frontend
+				if _, exists := templateObjects.Backends[defaultBackendName]; !exists {
+					// Ensure we are not duplicating the backend
+					templateObjects.Backends[defaultBackendName] = &types.Backend{
+						Servers: make(map[string]types.Server),
+						LoadBalancer: &types.LoadBalancer{
+							Method: "wrr",
+						},
+					}
+
+					service, exists, err := k8sClient.GetService(i.Namespace, i.Spec.Backend.ServiceName)
+					if err != nil {
+						log.Errorf("Error while retrieving service information from k8s API %s/%s: %v", i.Namespace, i.Spec.Backend.ServiceName, err)
+						return nil, err
+					}
+					if !exists {
+						log.Errorf("Service not found for %s/%s", i.Namespace, i.Spec.Backend.ServiceName)
+						continue
+					}
+
+					templateObjects.Backends[defaultBackendName].CircuitBreaker = getCircuitBreaker(service)
+					templateObjects.Backends[defaultBackendName].LoadBalancer = getLoadBalancer(service)
+					templateObjects.Backends[defaultBackendName].MaxConn = getMaxConn(service)
+					templateObjects.Backends[defaultBackendName].Buffering = getBuffering(service)
+
+					endpoints, exists, err := k8sClient.GetEndpoints(service.Namespace, service.Name)
+					if err != nil {
+						log.Errorf("Error retrieving endpoints %s/%s: %v", service.Namespace, service.Name, err)
+						return nil, err
+					}
+					if !exists {
+						log.Warnf("Endpoints not found for %s/%s", service.Namespace, service.Name)
+						break
+					}
+					if len(endpoints.Subsets) == 0 {
+						log.Warnf("Endpoints not available for %s/%s", service.Namespace, service.Name)
+						break
```

Comment:
If the service doest not exist, an `error` is logged and the `continue` instruction is called.
Can you explain to me why is there a `Warn` log and a `break` instruction when the `entrypoint.subset` does not exist please?